core/src/main/scala/akka/persistence/cassandra/Retries.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra
+
+import akka.actor.Scheduler
+import akka.annotation.InternalApi
+import akka.pattern.{ after, BackoffSupervisor }
+
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.concurrent.duration.FiniteDuration
+import scala.util.control.NonFatal
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[cassandra] object Retries {
```

Comment:
Simplified version of RetrySupport that also allows logging of message for each failure